### PR TITLE
added and tested visibility for service create and delete event with …

### PR DESCRIPTION
**Pull request by AkashDeepSrivastava**
…latest configurator controller routing logic,added dummy visibility module which can be removed later
@mak-454 @pkharat @njagadish @ashutosh-mishra @rajendramachani

### CODE REVIEW

**Comment by AkashDeepSrivastava** [comment]
Configurator config.py changes is added for log support for pecan server

File: gbpservice/nfp/configurator/api/v1/controllers/controller.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def post(self, **body):
                 body = pecan.request.json_body
 
             if self.method_name == 'network_function_event':
-                routing_key = 'EVENT'
+                routing_key = 'VISIBILITY'
```
Comment (pkharat) [inline gbpservice/nfp/configurator/api/v1/controllers/controller.py:137]: routing_key change should be done for PUT requests and also the comments block of _entry_to_rpc_routing_table() should be updated.

File: gbpservice/nfp/configurator/agents/loadbalancer_v1.py [hunk 2]
```diff
@@ -454,10 +472,12 @@ def _handle_event_vip(self, ev, operation):
                 driver.update_vip(old_vip, vip, context)
             elif operation == 'delete':
                 driver.delete_vip(vip, context)
+                self.plugin_rpc.vip_deleted(vip, lb_constants.ACTIVE, context)
```
Comment (pkharat) [inline gbpservice/nfp/configurator/agents/loadbalancer_v1.py:475]: vip delete status should be PENDING_DELETE right ?

**Comment by mak-454** [comment]
This is being reworked due to latest design changes.

**Pull request closed without merging**